src/Shared/FileMatcher.cs

```diff
@@ -1438,7 +1440,7 @@ enum GetSearchDataResult
                 needsRecursion);
 
             result.SearchData = searchData;
-            result.BaseDirectory = fixedDirectoryPart;
+            result.BaseDirectory = fixedDirectoryPart.TrimEnd(Path.AltDirectorySeparatorChar).TrimEnd(Path.DirectorySeparatorChar);
```

Comment:
Comment, please :)

---
In reply to: [75569935](https://github.com/Microsoft/msbuild/pull/924#discussion_r75569935) [](ancestors = 75569935)